manifests/cluster-selector-crd.yaml

```diff
@@ -18,36 +18,94 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.13.0
   name: clusterselectors.configmanagement.gke.io
   labels:
     configmanagement.gke.io/system: "true"
 spec:
   preserveUnknownFields: false
   group: configmanagement.gke.io
-  scope: Cluster
   names:
+    kind: ClusterSelector
+    listKind: ClusterSelectorList
     plural: clusterselectors
     singular: clusterselector
-    kind: ClusterSelector
+  scope: Cluster
   versions:
-    - name: v1
-      served: true
-      storage: true
-      schema:
-        # openAPIV3Schema is the schema for validating custom objects.
-        # TODO: Complete and tests this once 1.11 beta is out.
-        openAPIV3Schema:
-          type: object
-          required:
-          - spec
-          properties:
-            spec:
-              # ClusterSelectorSpec
-              type: object
-              required:
-              - selector
-              properties:
-                selector:
-                  type: object # metav1.LabelSelector
```

Comment:
I'm a bit confused by the previous CRD. What is the structure of the previous object? It doesn't look like it defines the same fields as the current API

Reply:
The previous CRD declares `x-kubernetes-preserve-unknown-fields: true` for this field, which doesn't expand the nested fields.

I don't know how was it initially generated. My best guess would be the CRD used to be v1beta1, and is now changed to v1/